test/k8sT/Policies.go

```diff
@@ -87,6 +86,38 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 			appPods   map[string]string
 		)
 
+		validateNoPolicy := func() {
+
+			ExpectWithOffset(1, kubectl.WaitCEPReady()).To(BeNil(), "Cep is not ready in a timeout")
+			for _, app := range apps {
+				pod := appPods[app]
+				cep := kubectl.CepGet(helpers.DefaultNamespace, pod)
+				ExpectWithOffset(1, cep).NotTo(BeNil(), "Endpoint %q does not exits", pod)
+
+				ExpectWithOffset(1, cep.Status.Policy.Realized.PolicyEnabled).To(
+					Equal(models.EndpointPolicyEnabledNone),
+					"Policy status does not match for endpoint %s", pod)
+			}
+		}
+
+		validatePolicyStatus := func() {
```

Comment:
Add small comments documenting what this function does; i.e., tests that app2 and app3 have no policy enforcement enabled, and that app1 has ingress policy enforcement enabled.

Reply:
done